src/Radio.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class Radio extends Component {
+    render() {
+
+        return (
+            <div className="form-group">
+                <input type={this.props.type} name={this.props.name} value="male" /> Male
```

Comment:
Why 'male', 'female', 'other' are harcoded? you need to pass them from the props